+ NUM_THREADS=1
+ SCRIPT=benchmark_scripts/connected_components-mpi/py/connected_components.py
+ MATRIX_PATH=matrices/amazon0601/amazon0601_ones.mtx
+ MATRIX_SIZE=403394
+ RESULT=data/mpi_scale_nodes/amazon0601/connected_components/py/4/1.dat
+ export OMP_NUM_THREADS=1
+ OMP_NUM_THREADS=1
+ export OPENBLAS_NUM_THREADS=1
+ OPENBLAS_NUM_THREADS=1
+ export MKL_NUM_THREADS=1
+ MKL_NUM_THREADS=1
+ export VECLIB_MAXIMUM_THREADS=1
+ VECLIB_MAXIMUM_THREADS=1
+ export NUMEXPR_NUM_THREADS=1
+ NUMEXPR_NUM_THREADS=1
+ export UCX_TLS=self,sm
+ UCX_TLS=self,sm
+ export OMPI_MCA_PML=ucx
+ OMPI_MCA_PML=ucx
+ export PMIX_MCA_gds=hash
+ PMIX_MCA_gds=hash
+ srun --mpi=pmix_v3 --cpus-per-task=1 singularity exec --env OMP_NUM_THREADS=1 --env PYTHONPATH=/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages /ceph/hpc/home/euretok/daphnesched2/jupycpp.sif python3 /ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py /ceph/hpc/home/euretok/daphnesched2/matrices/amazon0601/amazon0601_ones.mtx
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
WARNING: passwd file doesn't exist in container, not updating
WARNING: group file doesn't exist in container, not updating
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
srun: error: cn0174: tasks 35,43: Exited with exit code 1
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
srun: error: cn0174: tasks 32-34,36-42,44-63: Exited with exit code 1
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
Traceback (most recent call last):
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 127, in <module>
    cc(filename)
  File "/ceph/hpc/home/euretok/daphnesched2/benchmark_scripts/connected_components-mpi/py/connected_components.py", line 115, in cc
    x = G.multiply(c_partial.transpose()).max(axis=0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/yvqis8yj3dylnl4d7b4a75g9z8pxmh4i-python3-3.12.10-env/lib/python3.12/site-packages/scipy/sparse/_compressed.py", line 440, in multiply
    raise ValueError("inconsistent shapes")
ValueError: inconsistent shapes
srun: error: cn0183: tasks 64-95: Exited with exit code 1
srun: error: cn0186: tasks 96-127: Exited with exit code 1
srun: error: cn0173: tasks 0-31: Exited with exit code 1
